src/act/node/candid/variant/member.rs

```diff
@@ -3,38 +3,40 @@ use quote::quote;
 
 use super::Member;
 use crate::{
-    act::{node::CandidType, ToTypeAnnotation},
+    act::{
+        node::{CandidType, Context},
+        ToTypeAnnotation,
+    },
     keyword,
     traits::ToIdent,
 };
 
 impl Member {
     pub fn to_variant_member_token_stream(
         &self,
-        keyword_list: &Vec<String>,
+        context: &Context,
         parent_name: String,
     ) -> TokenStream {
         let member_type_token_stream = match self.candid_type.clone() {
             CandidType::Primitive(_) => {
                 if self
-                    .to_type_annotation(keyword_list, parent_name.clone())
+                    .to_type_annotation(context, parent_name.clone())
```

Comment:
Does it make sense to make the parent name part of the context? If so should we do it? If so should we maybe do it later?

Reply:
Probably not, right? Because a canister method would have a context, but it doesn't really have a parent name, or it could, it would just be the name of the service it was part of. It wouldn't have to use the info, if it didn't need to.

I'm just wondering if for the inline_name parameter it would help clear that up a little more. Anytime it needed the parent name it could pull that from the context, but if it needed more information, like the name of the parameter, then that kind of info would be required by list of parameter. There is just an awful lot of places where we ask for the parent name and then we don't use it, but we have to have it for all of the other things. So it kind of feels like the context a little.

Reply:
Though at the same time, how it is currently, once you make the context it's the same and you just pass it around everywhere. If we added the parent_name to the context, we would have to create a new context every time we went down a level in the act. If it's the more correct way to do it then maybe that's fine, or maybe it's even good. But if it's not the right or the wrong choice then that's a lot of unnecessary burden to pick up.